src/main/java/com/client/core/base/util/TriggerUtil.java

```diff
@@ -109,4 +92,9 @@ public static <E extends BullhornEntity> OneToMany<E> convertReplaceAllToEntityO
 		oneToMany.setTotal(bullhornEntities.size());
 		return oneToMany;
 	}
+
+	public static <E extends BullhornEntity> void setSpecialField(E entity, Map<String,Object> valuesChanged, String sourceField, String targetField){
```

Comment:
When is this method useful?

Reply:
... I have no idea where this came from. That is so weird. I'm going to remove this. Thanks for pointing it out

Reply:
Oh. I think it came from the cs-starter-template

Reply:
I'm tempted to remove it, I don't know why @cablke01 added this